### ISSUE TITLE
 Class XmlHelper is not compatible with webstart environment

### ISSUE THREAD

**Issue by kuniss** [JavaPOSWorkingGroup/javapos-config-loader#1]
[Migrated from sourceforge issue tracker, added by Anonymous: [https://sourceforge.net/p/jposloader/bugs/57/](https://sourceforge.net/p/jposloader/bugs/57/)]

The class `XmlHelper ` is not compatible with the webstart environment. There is referred a JRE system classloader instead of current classoader assigned to the class. The proposed correction does not affect non-webstart applications. See the code snippet below:

``` java
public void checkAndCreateTempDtd() {
is = ClassLoader.getSystemClassLoader().getResourceAsStream( dtdJarFullFileName );
}
```

should be:

``` java
is = getClass().getClassLoader().getResourceAsStream( dtdJarFullFileName );
```
The change needs to be done at _src/main/java/jpos/util/XmlHelper.java_.

**Comment by kuniss** [comment]
This can be tested with release candidate [RC2](https://github.com/JavaPOSWorkingGroup/javapos-config-loader/releases/tag/4.0.0-RC2).

Help for testing is very appreciated as I do not have an environment for this.

**Comment by kuniss** [comment]
This issue has been fixed in release [4.0.0](https://github.com/JavaPOSWorkingGroup/javapos-config-loader/releases/tag/4.0.0).
